app/decorator/song_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class SongDecorator < Draper::Decorator
+  def popularity
+    if object.plays > 3000
+      "#{object.name} has #{object.plays} plays, it is popular"
```

Comment:
Ah, so -- this won't happen due to your `> 3000` rule, but what if it had just one play.

How could you use this decorator to say "has 1 play"?

Hint: https://github.com/drapergem/draper#accessing-helpers